server_admin/topics/user-federation.adoc

```diff
@@ -2,45 +2,31 @@
+
+External user databases rarely have the data necessary to support all the features of {project_name}, so the User Storage Provider can opt to store items locally in {project_name} user data storage. Providers can import users locally and sync periodically with external data storage. This approach depends on the capabilities of the provider and the configuration of the provider.  For example, your external user data storage may not support OTP. The OTP can be handled and stored by {project_name}, depending on the provider.
 
-To add a storage provider go to the `User Federation` left menu item in the Admin Console.
+=== Adding a Provider
+To add a storage provider, perform the following procedure:
 
+.Procedure
+. Click *User Federation* in the menu.
++
 .User Federation
 image:{project_images}/user-federation.png[]
-
-On the center, there is an `Add Provider` list box.  Choose the provider type you want to add and you will be brought to the configuration page of that provider.
++
+. Click the *Add Provider* drop-down list.
+. Click the provider type you want to add, and {project_name} brings you to that provider's configuration page.
 
 === Dealing with Provider Failures
 
-If a User Storage Provider fails (for example, your LDAP server is down), you may have trouble logging in and may not be able to view users in the admin console.
-{project_name} does not catch failures when using a Storage Provider to lookup a user.  It will abort the invocation.  So, if you have a Storage Provider with a higher
-priority that fails during user lookup, the login or user query will fail entirely with an exception and abort.  It will not fail over to the next configured provider.
+If a User Storage Provider fails, you may not be able to log in and view users in the admin console. {project_name} does not detect failures when using a Storage Provider to lookup a user, so it cancels the invocation. If you have a Storage Provider with a high priority that fails during user lookup, the login or user query fails with an exception and will not fail over to the next configured provider.
 
-The local {project_name} user database is always searched first to resolve users before any LDAP or custom User Storage Provider.
-You may want to consider creating an admin account that is stored in the local {project_name} user database just in case any problems come up in connecting
-to your LDAP and custom back ends.
+{project_name} searches the local {project_name} user database first to resolve users before any LDAP or custom User Storage Provider. Consider creating an admin account stored in the local {project_name} user database in case of problems connecting to your LDAP and back ends.
```

Comment:
From the Bible: Refer to a user with administrator rights as an “administrator,” not an admin.